tests/unit/state/receiver/test_receiver_state.cpp

```diff
@@ -69,14 +73,15 @@ static std::vector<SacnMcastInterface> kTestNetints = {{{kEtcPalIpTypeV4, 1u}, k
                                                        {{kEtcPalIpTypeV4, 3u}, kEtcPalErrOk}};
 
 static const sacn_receiver_t kFirstReceiverHandle = 0;
-static const uint16_t kTestUniverse = 123u;
-static const uint8_t kTestPriority = 100u;
+static const uint8_t kTestPriority = 77u;
 static const bool kTestPreview = false;
 
 static const etcpal_socket_t kTestSocket = static_cast<etcpal_socket_t>(7);
 static TerminationSet kTestTermSet = {{0u, 0u}, {nullptr, nullptr, 0u, nullptr, nullptr, nullptr}, nullptr};
 
 static uint8_t test_data[SACN_MTU];
+static uint8_t seq_num;
+static unsigned int test_iteration;
```

Comment:
Good ideas. Whenever I resort to these static variables it's because I need them in the lambda functions. I'll think of some ways to bring these definitions closer to where they're actually used.

Reply:
Doing it the way you described works well - seq_num and test_data are static members, and test_iteration is a static member of the test itself. I'll try to follow this general approach in the future.

Reply:
I would think that in this case seq_num could even be a non-static member. test_data I'm not so sure about, it may need to be static still.

Reply:
Yeah, test_data is used by a lambda. I think the only reason seq_num is static is so that I could use it as a default value in `void InitTestData(uint8_t start_code, uint16_t universe, const uint8_t* data = nullptr, size_t data_len = 0u, uint8_t flags = 0u, uint8_t sequence_number = seq_num_)`